src/witan/app/test_data.clj

```diff
@@ -5,104 +5,147 @@
             [witan.app.model :as model]
             [witan.app.user :as u]))
 
-(def input-category-a  {:category "long population"
-                        :description "This a description of the long population input"})
+(def population-fixed-input  {:category "population"
+                              :description "Department of Communities and Local Government (DCLG) public population projections"})
+(def institutional-fixed-input {:category "institutional"
+                                :description "DCLG institutional population projections, also known as communal establishment pop (CEP) i.e. people living in student halls, retirement homes etc."})
+(def private-housing-fixed-input {:category "private-housing"
+                                  :description "DCLG household population projections i.e. people living in private rather than communal housing"})
+(def household-fixed-input {:category "households-data"
+                            :description "DCLG projections for numbers of households"})
+(def low-trend-fixed-input {:category "low-trend-data"
+                            :description "GLA-generated low migration scenario trend-based population projections"})
+(def high-trend-fixed-input {:category "high-trend-data"
+                             :description "GLA-generated high migration scenario trend-based population projections"})
+(def dwellings-data-fixed-input {:category "dwellings-data"
+                                 :description "Dwellings from most recent census"})
 
-(def input-category-b  {:category "overlay housing"
-                        :description "This a description of the overlay housing input"})
-
-(def input-category-c  {:category "trend data"
-                        :description "This a description of the trend data input"})
-
-(def output-category-a {:category "housing-linked population"
-                        :description "This a description of the housing-linked population output"})
+(def development-category {:category "development-data"
+                           :description "Net new dwellings from London Development Database and projections of housing (SHLAA or BPO)"})
+(def output-category {:category "housing-linked-population"
+                      :description "Housing-linked population projections"})
 
 (defn load-test-data!
-  "Add a bunch of test data into Cassandra"
+  "Loads realistic demo data into Cassandra"
```

Comment:
OK, fair enough.  we could create the demo data ad-hoc though when an actual demo is coming up, otherwise it'll just lie around waiting for attention.